BookingsApi/BookingsApi/Controllers/V2/HearingsControllerV2.cs

```diff
@@ -206,6 +206,8 @@ public async Task<IActionResult> UpdateHearingsInGroup(Guid groupId, [FromBody]
 
             var venues = await GetHearingVenues();
             
+            
```

Comment:
please can you remove these extra lines that were added?

Reply:
done